src/sentry/api/endpoints/organization_health.py

```diff
@@ -9,6 +9,7 @@
 from django.utils import timezone
 
 from sentry.api.bases import OrganizationEndpoint, EnvironmentMixin
+from sentry.api.endpoints.organization_events import InvalidParams, get_date_range_from_params
```

Comment:
We shouldn't really be doing this. Might wanna put these into some utils or something that's shared.